### ISSUE TITLE
[guideline]如何避免进度冲突？

### ISSUE THREAD

**Issue by shouldsee** [Kivy-CN/MLAPP-CN#16]
有没有类似分配表之类的防止重复翻译的机制？

**Comment by lhsfcboy** [comment]
目前还没有 可以利用readme中的翻译进度追踪的部分领取任务   楼主是从前向后翻译的

**Comment by shouldsee** [comment]
@lhsfcboy 是不是考虑做一个Issue？公共README编辑的权限比较复杂啊。

**Comment by lhsfcboy** [comment]
可以啊 不过估计楼主会直接给你添加编辑权限  等楼主上线 听听他的意见吧

**Comment by shouldsee** [comment]
@lhsfcboy 

明天要上班，“我等不到了”。 (@_@)

**Comment by lhsfcboy** [comment]
@cycleuser   有关翻译协作的具体办法, 有如下方案, 你看如何

翻译者提交Issue 格式如
Title: [认领翻译]第21章

认领翻译 第21章
译者：lhsfcboy
认领时间：2018/06/18
预计提交：2018/03/15

之后, 等待其pull request 或者 直接将翻译权限开放给译者
并且在readme.md的翻译进度追踪中加入相关认领信息, 例如:

```
- [ ] 21 Variational inference 731~766
  - lhsfcboy已经认领
```


如果这个方案可以的话, 就在readme.md中, 新加一节 协作翻译指南

**Comment by shouldsee** [comment]
@lhsfcboy 
建议在格式中区分小章节：
```
- [ ] 21 Variational inference 731~766
  -认领：shouldsee; 死线：180901 
    -章节: 21.1, 21.2,  21.5,  21.6;
  -认领：lhsfcboy; 死线：180901; 完成：180615（#3） 
    -章节： 21.3, 21.4; 
```
建议不要在README上直接记录，因为可能会很冗长，建议新开一个markdown文档(tasks.md或deadlines.md)，并在README中放入超链接。

**Comment by cycleuser** [comment]
具体细节你们商量着弄吧~
我不太懂这些~~
我能做的也就是每天按照顺序发个三两页的笔记~~

不过建议还是尽量每一章节的笔记由单人负责,避免出现一章多人,因为一章多人可能会导致同一章节内风格等方面的极大差异.

**Comment by cycleuser** [comment]
另外好像没必要弄Deadline之类的吧,那样弄太官僚了感觉. 区分小章节也太细了, 目前这种记录页面的方式既能避免冗长,也能避免过度复杂,感觉好像还能忍.

**Comment by shouldsee** [comment]
@cycleuser Deadline只是用来防止无限期拖延的....也就是说Deadline之后如果这个人没声音，大家就不用担心和他冲突了。当然这只是个大约摸的时间点跟官僚应该还扯不上关系。

至于风格问题我觉得问题不是特别大？后期统一校对一下调整一下风格就好了呗？因为我中间跳了一小节，所以光靠章节数字是无法记录这个信息的。。。而且这本书真的挺厚的啊

**Comment by cycleuser** [comment]
其实每章也就是几十页而已,因此还是建议尽量单人单章避免再次细分造成的碎片化,碎片化绝对是噩梦吖~~~ 

按照大约三十页一章来算,三十天足够了,如果三十天内没弄完,其他人接手就可以咯~ 咋样?

**Comment by shouldsee** [comment]
@cycleuser Yes that is the idea. Be sure to encourage each collaborator finish a whole chapter. But it's only something that to be expected not to be forced. 

(no chinese input on MacOS ATM)
